crawl-ref/source/dat/des/variable/ghost.des

```diff
@@ -0,0 +1,77 @@
+##############################################################################
+# Ghost vaults
+#
+##############################################################################
+
+###############################################################################
+# Two basic ghost vaults
+NAME:   basic_ghost_runed
+CHANCE: 25%
+DEPTH:  D:3-, Lair, Elf, Orc, Snake, Shoals, Swamp, Spider, Depths, Vaults, Crypt, Zot, Dis, Geh, Tar, Coc
+TAGS:   player_ghost luniq_player_ghost chance_player_ghost no_tele_into no_trap_gen allow_dup
+KFEAT:  + = runed_door
```

Comment:
The glyph `=` is already a runed door. I'd rather use that than this KFEAT on `+`.